lib/coarse_op_preconditioned.cu

```diff
@@ -82,8 +82,8 @@ namespace quda {
 #endif
         if (compute_max_only) {
           if (!activeTuning()) { // only do copy once tuning is done
-            qudaMemcpyAsync(arg.max_h, arg.max_d, sizeof(Float), cudaMemcpyDeviceToHost, stream);
-            qudaStreamSynchronize(const_cast<cudaStream_t&>(stream));
+            qudaMemcpyAsync(arg.max_h, arg.max_d, sizeof(Float), qudaMemcpyDeviceToHost, stream);
+            qudaStreamSynchronizeDriver(const_cast<qudaStream_t&>(stream));
```

Comment:
Why does have have `qudaStreamSynchronizeDriver` as opposed to `qudaStreamSynchronize`